<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrkManager</class>
 <widget class="QDialog" name="BrkManager">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Breakpoint editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QSpinBox" name="brkBank">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>bank</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>offset</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="xHexSpin" name="leStartOffset"/>
     </item>
     <item row="2" column="1">
      <widget class="xHexSpin" name="brkAdrHex">
       <property name="inputMask">
        <string>HHHH</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>abs</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>End adr</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Start adr:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Flags</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="brkFetch">
       <property name="text">
        <string>Fetch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="brkRead">
       <property name="text">
        <string>Read</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="brkWrite">
       <property name="text">
        <string>Write</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="xHexSpin" name="leEndOffset"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>offset</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="xHexSpin" name="brkAdrEnd"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>abs</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="1">
    <widget class="xHexSpin" name="brkMaskHex">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="inputMask">
      <string>HHHH</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="Line" name="line_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOK">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="../xpeccy.qrc">
         <normaloff>:/images/ok-apply.png</normaloff>:/images/ok-apply.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>IO Mask</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="brkType"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xHexSpin</class>
   <extends>QLineEdit</extends>
   <header location="global">stdlib.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../xpeccy.qrc"/>
 </resources>
 <connections/>
</ui>
